src/test/java/seedu/mtracker/console/AddCryptoParserTest.java

```diff
@@ -55,11 +52,6 @@ void addCryptoParams_noRemarks_expectSuccess() {
         testCryptoParameters(USER_INPUT_NO_REMARKS, EXPECTED_PARAMS_NO_REMARKS);
     }
 
-    @Test
-    void addCryptoParams_noExpiry_expectSuccess() {
-        testCryptoParameters(USER_INPUT_NO_EXPIRY, EXPECTED_PARAMS_NO_EXPIRY);
-    }
-
```

Comment:
Nice work, clearing redundant dead code. Btw can we add a tryNoExpiryMultipletimes test